src/tests/Notes.test.js

```diff
@@ -0,0 +1,33 @@
+import { render } from '@testing-library/react';
+import { GlobalContext } from '../context/GlobalContext';
+import { BrowserRouter } from 'react-router-dom';
+import Notes from '../pages/private/Notes';
+import PrivateRoute from '../components/PrivateRoute';
+
+describe('Notes', () => {
```

Comment:
I think for this one and the Archives there should be a test that makes sure you are rendering correctly the notes that are there.